File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/EOSGrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;



import java.io.Serializable;

public class EOSGrunnlag implements Serializable {

    /**
     * Foreløbig - skal ikke brukes
     */


    /**
     * Garantibeløp for alderspensjons
     */
    private int apgBelop;

    /**
     * faktisk trygdetid i Norge, i måneder
     */
    private int faktiskTT_Norge;

    /**
     * faktisk trygdetid i Norden, utenom Norge, i måneder
     */
    private int faktiskTT_Norden;

    /**
     * faktisk trygdetid i EØS land, i måneder
     */
    private int faktiskTT_EOS;

    /**
     * faktisk trygdetid i øvrige land med bilateral avtale, i måneder
     */
    private int faktiskTT_Ovrige;

    /**
     * Antall faktiske trygdetidsår - brukes ved bilaterale avtale med Canada
     */
    private int faktiskTT_Ar;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/EOSGrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable

class EOSGrunnlag : Serializable {
    /**
     * Garantibeløp for alderspensjons
     */
    var apgBelop = 0

    /**
     * faktisk trygdetid i Norge, i måneder
     */
    var faktiskTT_Norge = 0

    /**
     * faktisk trygdetid i Norden, utenom Norge, i måneder
     */
    var faktiskTT_Norden = 0

    /**
     * faktisk trygdetid i EØS land, i måneder
     */
    var faktiskTT_EOS = 0

    /**
     * faktisk trygdetid i øvrige land med bilateral avtale, i måneder
     */
    var faktiskTT_Ovrige = 0

    /**
     * Antall faktiske trygdetidsår - brukes ved bilaterale avtale med Canada
     */
    var faktiskTT_Ar = 0

}